include/deepworks/shape.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <vector>
+
+namespace deepworks {
+
+using Shape = std::vector<int>;
```

Comment:
Please rebase your branch to master

Reply:
Move `Shape` to another file, to use it from placeholder